### DISCUSSION TITLE
Prisma Postgres triggers support

### DISCUSSION THREAD

**Discussion by ggukggukee** [prisma/prisma#14629] [Q&A] [answered]
Is Prisma already support Postgres triggers for table changes or we have to use Postgres transactions for that?

I want history/logs for my data changes in tables. But heard that Prisma is not supporting triggers.

**Comment by nurul3101** [comment, marked as answer]
Hey @ggukggukee 👋 

Prisma doesn't have support for real-time database event triggers yet. We do have a Feature Request for it: [#905](https://github.com/prisma/prisma/issues/905)
Could you use [Middleware](https://www.prisma.io/docs/concepts/components/prisma-client/middleware) to log data changes anytime a query is executed?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
